tests/test_stackdriver_log_formatter.py

```diff
@@ -69,8 +69,7 @@ def test_info(logger_and_data: LoggerAndData) -> None:
 
 def test_info_with_custom_data(logger_and_data: LoggerAndData) -> None:
     logger, data = logger_and_data
-    logger.info('I have a data: %(value)d',
-                {'value': 99, 'module': 'You cannot override!', 'stackInfo': 'Cannot override too'})
+    logger.info('I have a data: %(value)d', {'value': 99, 'module': 'cannot override', 'stackInfo': 'cannot override'})
```

Comment:
[note]
On python <3.8, it's line number reported as 73 (where the function-call paren ends), while it's 72 (where the paren begins) on python 3.8.
Such difference is not something we should care, that's why I modified test case so that function call does not spans multiple lines.